util/src/strings.cc

```diff
@@ -6,7 +6,7 @@
 namespace leatherman { namespace util {
 
     std::string plural(int num_of_things) {
-        return num_of_things > 1 ? "s" : "";
+        return num_of_things == 1 ? "" : "s";
```

Comment:
If we're going to bother to pluralize, we might as well be grammatically correct: "0 things", not "0 thing".